src/main/resources/application.yml

```diff
@@ -72,3 +88,37 @@ app:
 spring:
   profiles: auth
 
+---
+###############################################################################
+# Profile - "debug"
+###############################################################################
```

Comment:
maybe personal, I think the debug profile is not needed because those values are already in docker-compose.yml.

Reply:
its very useful, because it allows me to develop in my IDE, and jsut run ego. This way i can set break points and build quickly. I could build the entire docker image for program service and then open a debug port and hook up the ide, but this is just less complicated.

Reply:
I still personally think it's simpler to make the code simple with a little sacrifice of a little bit more code changes. But I won't block it because I think it might be personal